langchain/src/retrievers/index.ts

```diff
@@ -10,3 +10,4 @@ export {
 } from "./supabase.js";
 export { RemoteLangChainRetriever } from "./remote/remote-retriever.js";
 export { MetalRetriever } from "./metal.js";
+export { VespaRetriever } from "./vespa.js";
```

Comment:
We're deprecating all the `index.ts` files, wouldn't cause a problem in this case but best not to include it here.